README.md

```diff
@@ -17,12 +17,23 @@ Requirements
 ============
 
 Supported versions:
-* Python 2.6
+* Python 2.6 (currently not supported on AppVeyor)
```

Comment:
2.6 is only really supported by RHEL these days. I don't mind dropping support for it.